lib/config/rules/functionality.js

```diff
@@ -0,0 +1,80 @@
+// Functionality Specific:
+// These rules catch common errors in JS programming or
+// otherwise confusing constructs that are prone to producing bugs:
+// Reference https://palantir.github.io/tslint/rules/
+
+module.exports = {
+  // Bans the use of specific functions or global methods.
+  'ban': false,
```

Comment:
If we turned off the rule in the config, that would turn it off for **all** projects that use the config. For project-specific configuration, we rely on extending our "ideal" config and adding the few overrides we need for that project to only its own config.